### ISSUE TITLE
Credhub login

### ISSUE THREAD

**Issue by ranjithfd** [pivotal-cf/credhub-release#36]
What version of the credhub server you are using?
Standalone bosh

What version of the credhub cli you are using?

2.2

What did you expect to happen?

I need to login and store all the credential. 

What was the actual behavior?

 ./credhub login --server https://*********.com:8844/api --skip-tls-validation
username: admin
password: ********************
invalid_token: An I/O error occurred while reading from the JWK Set source: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target

**Comment by tomkennedy513** [comment]
Hi, can you you provide a little more information about your environment? Also, what version of CredHub server are you running?

**Comment by ranjithfd** [comment]
Hi Tom,

           We are trying to install credhub on  standalone bosh director. 

UAA is already pre-configured in our PAS (ERT) environment, we are planning to use the existing UAA with new credhub server. 

 credhub               credhub/2.2.0                           
 bosh                    vsphere-esxi-ubuntu-xenial-go_agent/170.12   -
  fim              -       1.3.4
 postgres             30

**Comment by ranjithfd** [comment]
HI Team,

          Any input on this.

**Comment by ankeesler** [comment]
@ranjithfd - thanks for your patience. We noticed that you referenced a commercial Cloud Foundry offering. We would recommend going through the support channels for that particular commercial offering instead of using the open source GitHub repository. We believe that you will reach a faster resolution to your issue through the PAS support channels.

That being said, we can try to offer assistance here.  I noticed that your command above contains `/api` in the server address: `./credhub login --server https://*********.com:8844/api --skip-tls-validation`. Did you try removing the `/api` from that server address? Perhaps you have a custom setup for your CredHub, but I am used to being able to access CredHub using the root path, i.e., `./credhub login --server https://***.com:8844 --skip-tls-validation`.
